File: Java/CheckOutApp.java
import java.util.ArrayList;
import java.util.Scanner;


public class CheckOutApp{

public static void main(String...args){

ArrayList<String> itemNames = new ArrayList<>();
ArrayList<Integer> itemPrices = new ArrayList<>();
ArrayList<Integer> itemQuantities = new ArrayList<>();
ArrayList<Integer> itemTotalAmount = new ArrayList<>();
Scanner input = new Scanner(System.in);


System.out.print("What is the Customer name: ");
String customerName = input.next();

int loopCondition = 0;

while(loopCondition!=1){
System.out.print("\nWhat is the Item Name: ");
String itemName = input.next();
System.out.print("What is the Item Price: ");
int itemPrice = input.nextInt();
System.out.print("What is the Item Quantity: ");
int itemQuantity = input.nextInt();

System.out.println("Do you wish to add another item (0-No, Any Other Number - Yes): ");
int addItem = input.nextInt();
CheckOutAppFunctions.addItemNameToItemNamesList(itemName, itemNames);
CheckOutAppFunctions.addItemPriceToItemPriceList(itemPrice, itemPrices);
CheckOutAppFunctions.addItemQuantityToItemQuantityList(itemQuantity, itemQuantities);

switch(addItem){
	case 0:
		loopCondition = 1;
		break;
	default:
		System.out.println("\nProceed to add another item\n");

}

}

System.out.print("What is your name: ");
String cashierName = input.next();

System.out.print("What is the discount: ");
int discountInPercentage = input.nextInt();
itemTotalAmount = CheckOutAppFunctions.getTotalAmountArrayList(itemTotalAmount, itemPrices, itemQuantities);

CheckOutAppFunctions.printInvoice(cashierName, customerName, itemNames, itemPrices, itemQuantities, itemTotalAmount, discountInPercentage);

System.out.print("How much did the customer pay: NGN");
int AmountPaidByCustomer = input.nextInt();

CheckOutAppFunctions.printReceipt(cashierName, customerName, itemNames, itemQuantities, itemPrices, itemTotalAmount, discountInPercentage, AmountPaidByCustomer);

}
}

File: python/checkoutapp.py
from checkoutfunctions import *
list_of_item_names = []
list_of_item_prices = []
list_of_item_quantity = []


customer_name = input("What is the customer's Name: ")

while True:
	item_name = input("What did customer buy? ")
	add_item_name_to_list_of_item_names(item_name, list_of_item_names)
	item_quantity = int(input("How many pieces? "))
	add_item_quantity_to_list_of_item_quantities(item_quantity, list_of_item_quantity)
	item_prices = int(input("How much per unit? "))
	add_item_prices_to_list_of_item_prices(item_prices, list_of_item_prices)
	loop_condition = input("Add more items? ")
	match(loop_condition):
		case "no":
			break
cashier_name = input("What is your name? ")
discount = int(input("How much discount will the customer get: "))

print_invoive(cashier_name, customer_name, list_of_item_names, list_of_item_prices, list_of_item_quantity, discount)


amount_paid = int(input("How much did the customer give to you? "))


print_receipt(cashier_name, customer_name, list_of_item_names, list_of_item_prices, list_of_item_quantity, discount, amount_paid)





			


